src/App.js

```diff
@@ -1,9 +1,40 @@
+// Import react module and components
 import React from 'react';
+import Button from './components/Button';
+import Product from './components/Product';
+import Section from './components/Section';
+// Import assets
 import './App.css';
+import bag_1 from './assets/bag_1.png';
+import bag_2 from './assets/bag_2.png';
+import bag_3 from './assets/bag_3.png';
+import bag_4 from './assets/bag_4.png';
+import brand from './assets/brand.png';
+import our_story from './assets/our_story.png';
 
 function App() {
   return (
+    <>
       <h1>Handbags & Purses</h1>
+      <nav>
+        <Button link='#' text='To the collection' />
+        <Button link='#' text='Shop all bags' />
+        <Button link='#' text='Pre-orders' isDisabled={true}/>
```

Comment:
Voor het consequent schrijven van de code zou ik de `isDisabled={true}` ook plaatsen in de ander buttons, maar dan uiteraard {false}